ckanext/schema/dc_dataset.yaml

```diff
@@ -0,0 +1,321 @@
+    required: false
+    preset: date
+    display_property: dcat:startDate
+    help_text: Earliest date the dataset covers e.g. 2016-01-01.
+    
+  # Will be used to build a filter (e.g. by year)
+  - field_name: temporal_end
+    label: End date
+    required: false
+    preset: date
+    display_property: dcat:endDate
+    help_text: Latest date the dataset covers e.g. 2019-01. Leave blank if dataset is updated to present day.
+
+  # Will be used to build a filter
+  - field_name: temporal_resolution
+    label: Time resolution
+    required: false
+    preset: select
+    choices:
+    - value: sub_second
+      label: Sub-second
+    - value: second
+      label: Second
+    - value: minute
+      label: Minute
+    - value: hour
+      label: Hour
+    - value: day
+      label: Day
+    - value: week
+      label: Week
+    - value: month
+      label: Month
+    - value: quarter
+      label: Quarter
+    - value: year
+      label: Year
+    display_property: dcat:temporalResolution
+    help_text: For time-series data, please select the smallest resolution represented in your dataset. If your dataset resolution falls between the values given here, please select the closest resolution (e.g. for readings every 30 seconds select minute).
+  
+  # Could be used to automate communication with publisher to update their dataset
+  # Could also feed into freshness or future data quality scores
+  - field_name: update_frequency
+    label: Update frequency
+    required: false
+    preset: select
+    choices:
+    - value: hourly
+      label: Hourly
+    - value: daily
+      label: Daily
+    - value: weekly
+      label: Weekly
+    - value: monthly
+      label: Monthly
+    - value: quarterly
+      label: Quarterly
+    - value: annually
+      label: Annually
+    - value: never
+      label: Never
+    display_property: dcat:accrualPeriodicity
+    help_text: Select the closest value that reflects the frequency with which the dataset is updated.
+
+  # Will be used to build a filter
+  - field_name: iso_region
+    label: ISO region
+    required: false
+    preset: select
+    choices:
+    - value: emea
+      label: EMEA
+    - value: apac
+      label: APAC
+    - value: americas
+      label: AMERICAS
+    - value: world
+      label: WORLD
+    display_property: dct:Location
+    help_text: Select the broad region that your dataset covers.
+
+  # Will be used to build a filter
+  # TODO create scheming_countries_choices helper (see: https://github.com/ckan/ckanext-scheming#choices_helper)
+  - field_name: countries
+    label: Countries
+    required: false
+    preset: multiple_select
+    choices_helper: scheming_countries_choices
+    display_property: dct:Location
+    help_text: Select the countries that your dataset covers.
+
+  # Will be used to build a filter
+  - field_name: spatial_resolution
+    label: Spatial resolution
+    required: false
+    preset: select
+    choices:
+    - value: 1m-grid
```

Comment:
what about e.g.
vehicle
person
company
parent company
industry / sector
fuel type

I feel like the satellite resolution is out of place with the other artifacts as discussed. Then again this seems like a reasonable solution. 
We could always allow an 'Other' - custom field and then check to see how often it is being used.

We might want to separate 'asset' out as 'asset-level data' is a big theme for e.g. better financial data

Reply:
Hmm, yes this is a difficult one - I tried to stick to geographic terms for the most part and added `device/asset` for anything below building level - perhaps that could be removed. I wouldn't expect people to use this field for things like fuel type or industry (these are more theme-level ideas and would be better suited as a tag). I guess the goal of this field is not to determine the aggregation type, just simply the minimum spatial resolution of the data. If a dataset is about (for example) the share price of a list of green tech companies, I wouldn't expect they'd use this field at all, as the data isn't spatially mapped in any particular way.

Reply:
That does make sense, but highlights the limits of just having temporal and spatial resolution. 
I'm content with it for now though.
We could have a specific tag for `asset-level`

Reply:
👍